.github/workflows/testing.yml

```diff
@@ -2,7 +2,7 @@ name: generic-repodispatch
 
 on:
   repository_dispatch:
-    types: [run-completed]
+    types: [run-completed1]
```

Comment:
run-completed1 is not an event I suppose

Reply:
Not a event.Changed the name so that this workflow does not trigger.

Reply:
just comment anything below this.. you just need to write a wf, it is ok if it is empty